va/va.go

```diff
@@ -93,12 +101,13 @@ func New(
 	clk clock.Clock,
 	httpPort, tlsPort int) *VAImpl {
 	va := &VAImpl{
-		log:      log,
-		clk:      clk,
-		httpPort: httpPort,
-		tlsPort:  tlsPort,
-		tasks:    make(chan *vaTask, taskQueueSize),
-		sleep:    true,
+		log:       log,
+		clk:       clk,
+		httpPort:  httpPort,
+		tlsPort:   tlsPort,
+		tasks:     make(chan *vaTask, taskQueueSize),
+		sleep:     true,
+		sleepTime: 15,
```

Comment:
Can you make a `const` for the default value? Thanks!

Reply:
Done. I called it `defaultSleepTime`; if you want the `default` removed from the name, tell me.